config/config-inyo/config_ignore.settings.yml

```diff
@@ -1,6 +1,7 @@
 ignored_config_entities:
   - openid_connect.settings.windows_aad
   - geocoder.geocoder_provider.bingmaps
+  - google_tag.settings.yml
```

Comment:
The config name should be without the "yml" extension.

Reply:
Fixed